plugins/eddn.py

```diff
@@ -424,51 +570,64 @@ def filter_localised(d):
         # add mandatory StarSystem, StarPos and SystemAddress properties to Scan events
         if 'StarSystem' not in entry:
             if not system:
-                return("system is None, can't add StarSystem")
+                return "system is None, can't add StarSystem"
+
             entry['StarSystem'] = system
+
         if 'StarPos' not in entry:
             if not this.coordinates:
-                return("this.coordinates is None, can't add StarPos")
+                return "this.coordinates is None, can't add StarPos"
+
             entry['StarPos'] = list(this.coordinates)
+
         if 'SystemAddress' not in entry:
             if not this.systemaddress:
-                return("this.systemaddress is None, can't add SystemAddress")
+                return "this.systemaddress is None, can't add SystemAddress"
+
             entry['SystemAddress'] = this.systemaddress
 
         try:
             this.eddn.export_journal_entry(cmdr, is_beta, filter_localised(entry))
+
         except requests.exceptions.RequestException as e:
             logger.debug('Failed in export_journal_entry', exc_info=e)
             return _("Error: Can't connect to EDDN")
+
         except Exception as e:
             logger.debug('Failed in export_journal_entry', exc_info=e)
             return str(e)
 
     elif (config.getint('output') & config.OUT_MKT_EDDN and not state['Captain'] and
-          entry['event'] in ['Market', 'Outfitting', 'Shipyard']):
+            entry['event'] in ('Market', 'Outfitting', 'Shipyard')):
+
         try:
             if this.marketId != entry['MarketID']:
                 this.commodities = this.outfitting = this.shipyard = None
                 this.marketId = entry['MarketID']
 
-            with open(join(config.get('journaldir') or config.default_journal_dir, '%s.json' % entry['event']), 'rb') as h:
+            with open(
+                join(str(config.get('journaldir') or config.default_journal_dir), f'{entry["event"]}.json'), 'rb'
```

Comment:
I'm surprised you're not pathlib'ing things like this.

Reply:
was trying not to make that sort of change, but if Im good to do that I very much will